getting_started/workflow/export/exporting_for_android.rst

```diff
@@ -12,11 +14,15 @@ Download the Android SDK
 Download and install the Android SDK from
 https://developer.android.com/studio/
 
+- You need to run Android Studio once to complete the setup.
+
 Install OpenJDK or Oracle JDK
 -----------------------------
 
 Download and install  `OpenJDK <https://github.com/ojdkbuild/ojdkbuild>`__ or `Oracle JDK <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`__. Versions below JDK 8 may not work; some users have reported issues with the jarsigner (used to sign the APKs) in JDK 7.
 
+- If you install OpenJDK, choose ```1.8```. and choose do not choose the ```openjdk-jre``` files as that is only the JRE, not the JDK.
```

Comment:
```suggestion
- If you install OpenJDK, choose ``1.8``. Don't choose the ``openjdk-jre`` files as that only contains the JRE, not the JDK which is required here.
```

Reply:
looks good, I don't know how to apply your suggested change to my pull request.  maybe I cannot (only members of godot-docs can?)